src/v/raft/heartbeat_manager.cc

```diff
@@ -44,49 +49,58 @@ static std::vector<heartbeat_manager::node_heartbeat> requests_for_range(
     if (c.empty()) {
         return {};
     }
+
+    // Set of follower nodes whose heartbeat_failed status indicates
+    // that we should tear down their TCP connection before next heartbeat
+    absl::flat_hash_set<model::node_id> reconnect_nodes;
```

Comment:
pre-aalloc the size to some average ?

Reply:
Under normal operation the size will be zero, so avoiding allocating at all until we see a follower in a bad state.

Reply:
ack